src/padStart.js

```diff
@@ -0,0 +1,25 @@
+import padCharsStart from './padCharsStart';
+
+/**
+ * The function pads the current string with an empty string
```

Comment:
What do you say to the following: `Pads string on the left side if it's shorter than length. Padding characters are truncated if they exceed length.`

Reply:
Thanks for the feedback! One clarification on this bit:

> Padding characters are truncated if they exceed length.

What would be an example of this happening?

Reply:
You're right! We can drop this last sentence, because that is specific to `padCharsStart` and not to `padStart`

Reply:
Ok, cool. I was concerned the implementation was missing something after reading your initial comment 😅. Will update shortly. Thanks again!